src/adapter.ts

```diff
@@ -29,6 +29,38 @@ export class PrismaAdapter implements Adapter {
     }
   }
 
```

Comment:
missed filter related stuff, search "filtered" in: https://github.com/node-casbin/sequelize-adapter/pull/64/files